src/corefx/System.Security.Cryptography.Native/openssl.c

```diff
@@ -623,3 +655,123 @@ GetX509NameInfo(
 
     return NULL;
 }
+
+/*
+Function:
+GetX509StackFieldCount
+
+Used by System.Security.Cryptography.X509Certificates' OpenSslX509ChainProcessor to identify the
+number of certificates returned in the built chain.
+
+Return values:
+0 if the field count cannot be determined, or the count of certificates in STACK_OF(X509)
+Note that 0 does not always indicate an error, merely that GetX509StackField should not be called.
+*/
+int
+GetX509StackFieldCount(
+    STACK_OF(X509)* stack)
```

Comment:
A typed SafeHandle, actually; but yeah.

I could expand it here to "struct stack_st_X509", but STACK_OF(X509) is how it's always referenced